.github/workflows/build.yml

```diff
@@ -121,6 +121,8 @@ jobs:
       continue-on-error: ${{ matrix.rust-toolchain == 'nightly' }}
       run: make PLATFORM=aarch64-bsta1000b A=examples/helloworld SMP=8
 
+    - name: Install u-boot-tools for aarch64-phytium-pi
+      run: sudo apt update; sudo apt install -y u-boot-tools
```

Comment:
Why we need to install `u-boot-tools` in CI?

Reply:
CI will using mkimage tool to compile u-boot bootable image: arceos.uImage

Reply:
After adjusting makefile, no longer need to modify the CI build.yml